spec/spec_helper.rb

```diff
@@ -2,14 +2,7 @@
   require 'simplecov'
   require 'coveralls'
 
-  if ENV['TRAVIS']
```

Comment:
Should it be `ENV['CI']` instead?

Reply:
The branch hasn't been run the whole time we've been using GitHub Actions. If we want a different coverage output format in the future we can pick it then.